src/components/Editor/EditorMenu.js

```diff
@@ -151,6 +152,18 @@ function getMenuItems(
     click: () => addExpression(editor.codeMirror.getSelection())
   };
 
+  const executeInConsoleItem = {
+    id: "node-menu-execute-in-console",
+    label: executeInConsoleLabel,
+    click: () => {
+      new Promise((res, rej) => {
+        res(eval(selectionText));
```

Comment:
we probably want to the action `evaluateExpression("console.log(${selectionText})")` so that we have the server try to evaluate the expression `console.log(foo)` or whatever